src/components/LoanCards/Buttons/LendAmountButton.vue

```diff
@@ -25,10 +25,15 @@ export default {
 			default: false
 		},
 		amountLeft: {
-			type: Number,
+			type: [Number, String],
 			default: 20
 		}
 	},
+	computed: {
+		amountValue() {
+			return parseFloat(this.amountLeft);
```

Comment:
I think you'll need to use parseInt here so the decimal places are removed.

Reply:
Fixed!